### ISSUE TITLE
Add a contributing guide. 

### ISSUE THREAD

**Issue by Andrew-Pynch** [AnEmortalKid/sc-refinery-tracker#27]
I would like to make a PR with a couple of features I have in mind for your review, but can't build this application.

**Comment by AnEmortalKid** [comment]
Ah yeah, good idea.

I'll write some details here and then transpose them to a contributions guide.



This project uses [npm](https://www.npmjs.com/) and [webpack](https://webpack.js.org/) to end up producing a minified javascript file and single page app:

```
$ ls -l dist/
total 220
drwxr-xr-x 1 NeoSteelFrame 197121      0 Jan 23  2021 fonts/
-rw-r--r-- 1 NeoSteelFrame 197121  14765 Nov 24 17:27 index.html
-rw-r--r-- 1 NeoSteelFrame 197121 201726 Nov 24 17:27 main.js+++
```

I usually test things out from the index.html in the `dist` folder. 

The index.html actually includes a bunch of "partial" or chunks of html: https://github.com/AnEmortalKid/sc-refinery-tracker/blob/main/src/app.html#L10 (i wasn't sure how to structure things, since I don't usually make web-apps).  This is done with the HMTL plugin for webpack [here](https://github.com/AnEmortalKid/sc-refinery-tracker/blob/main/webpack.config.js#L13)

Let me know if there's any other questions!

**Comment by Andrew-Pynch** [comment]
That's really awesome, definitely turned out good since the app is pretty solid 👍 

I'll try compiling it using the steps you described.

**Comment by Andrew-Pynch** [comment]
#28

**Comment by AnEmortalKid** [comment]
Fixed by #28
